compiler-rt/lib/builtins/cpu_model/x86.c

```diff
@@ -512,15 +523,15 @@ static const char *getIntelProcessorTypeAndSubtype(unsigned Family,
     // Alderlake:
     case 0x97:
     case 0x9a:
+    // Gracemont:
```

Comment:
Why is 0xbe moved above?

Reply:
That's how it was on the LLVM side, and I figured moving it into alphabetical order by codename made more sense than sticking it at the end.

No real preference here though. The only thing I'm trying to get is consistency between the two implementations.

Reply:
@FreddyLeaf where's the Gracemont model from? I didn't find it in ISE. Besides, the Gracemont is not CPU name but microarchitecture name.